aviatrix/resource_spoke_vpc_test.go

```
@@ -33,6 +33,15 @@ func TestAccAviatrixSpokeGw_basic(t *testing.T) {
 
 	preGatewayCheck(t, msgCommon)
```

Might need to create one more function to throw error if required variables are not set